easybuild/easyconfigs/p/PSORTb/PSORTb-3.0.4_CC.patch

```diff
@@ -0,0 +1,13 @@
+#Subtitue hardcoded cc for gcc to prevent 'cc1: error: unrecognized command line option "-fstack-protector-strong"'
+#Author: Ruben van Dijk (University of Groningen)
+--- bio-tools-psort-all/bio-tools-psort-modhmm/Makefile	2016-08-29 09:40:30.000000000 +0200
++++ bio-tools-psort-all/bio-tools-psort-modhmm/Makefile	2016-08-29 09:40:14.000000000 +0200
```

Comment:
@RvDijk the patch doesn't work because the `Makefile` file is generated at configure time...

Two options:

* patch `Makefile.PL` instead, which involves grabbing the value of `$CC` in Perl from the environment rather than hardcoding `cc`
* do a runtime patch of `Makefile` via `prebuildopt` using a `sed` command to change the harcoded `CC =cc` to `CC ?= cc`